js/h5p-tooltip.js

```diff
@@ -0,0 +1,149 @@
+/*global H5P*/
+H5P.Tooltip = (function (EventDispatcher) {
+  "use strict";
+
+  /**
+   * Create an accessible tooltip
+   * 
+   * @param {HTMLElement} triggeringElement The element that should trigger the tooltip
+   * @param {Object} options Options for tooltip
+   * @param {String} options.text The text to be displayed in the tooltip
+   * @param {String[]} options.classes Extra css classes for the tooltip
+   * @param {Boolean} options.ariaHidden Whether the hover should be read by screen readers or not
+   * @constructor
+   */
+  function Tooltip(triggeringElement, options) {
+    EventDispatcher.call(this);
+    
+    /** @alias H5P.Tooltip */
+    let self = this;
+
+    // Make sure tooltips have unique id
+    H5P.Tooltip.uniqueId += 1;
+    const tooltipId = "h5p-tooltip-" + H5P.Tooltip.uniqueId;
+
+    // Default options
+    options = options || {};
+    options.text = options.text || '';
+    options.classes = options.classes || [];
+    options.ariaHidden = options.ariaHidden || false;
+
+    // Initiate state
+    this.hover = false;
+    this.focus = false;
+
+    // Declare constants for event listeners
+    const iframeBody = document.getElementsByTagName('body')[0];
+
+    const escapeFunction = function (e) {
+      if (e.key === 'Escape') {
+        tooltip.classList.remove('h5p-tooltip-visible');
+      }
+    }
+
+    // Create element
+    const tooltip = document.createElement('div');
+
+    tooltip.classList.add('h5p-tooltip');
+    tooltip.id = tooltipId;
+    tooltip.role = 'tooltip';
+    tooltip.innerHTML = options.text;
+    tooltip.ariaHidden = options.ariaHidden;
+    options.classes.forEach(extraClass => {
+      tooltip.classList.add(extraClass);
+    });
+
+    triggeringElement.appendChild(tooltip);
+
+    // Aria-describedby will override aria-hidden
+    if (!options.ariaHidden) {
+      triggeringElement.setAttribute('aria-describedby', tooltipId);
+    }
+    
+    //Add event listeners to triggeringElement
+    triggeringElement.onmouseenter = function () {
```

Comment:
This should be done using addEventListener instead, at least because of consistency. I.e:
`triggeringElement.addEventListener('mouseenter', ...)`